From: Mike Hommey <mh@glandium.org>
Date: Sat, 18 Oct 2008 14:30:06 +0200
Subject: Remove general.useragent.locale prefs from firefox*.js

It is better defined in the GRE provided by xulrunner-1.9
---
 browser/app/profile/firefox.js        | 1 -
 browser/locales/en-US/firefox-l10n.js | 1 -
 2 files changed, 2 deletions(-)

diff --git a/browser/app/profile/firefox.js b/browser/app/profile/firefox.js
index 34c93be..ac66958 100644
--- a/browser/app/profile/firefox.js
+++ b/browser/app/profile/firefox.js
@@ -250,7 +250,6 @@ pref("browser.uitour.whitelist.add.260", "www.mozilla.org,support.mozilla.org");
 
 pref("keyword.enabled", true);
 
-pref("general.useragent.locale", "@AB_CD@");
 pref("general.skins.selectedSkin", "classic/1.0");
 
 pref("general.smoothScroll", true);
diff --git a/browser/locales/en-US/firefox-l10n.js b/browser/locales/en-US/firefox-l10n.js
index 642ad65..b410ad9 100644
--- a/browser/locales/en-US/firefox-l10n.js
+++ b/browser/locales/en-US/firefox-l10n.js
@@ -4,4 +4,3 @@
 
 #filter substitution
 
-pref("general.useragent.locale", "@AB_CD@");
